main/src/ca/uwaterloo/flix/language/phase/Parser2.scala

```diff
@@ -2510,6 +2511,7 @@ object Parser2 {
       } else {
         // Case 3: new Type(exps...)
         arguments()
+        arguments()
```

Comment:
What? Explain... You call arguments twice???

Reply:
ah, my bad. this also shows why `arguments` behave a bit funny. it fails very silently